task/testmap.py

```diff
@@ -140,16 +140,15 @@
             'fedora-30/edge',
             'fedora-31/firefox',
             'rhel-7-8/firefox',
-            'rhel-8-1/chrome',
+            'rhel-8-2/chrome',
         ],
-        'rhel-8.1': [
-            'rhel-8-1/chrome',
-            'rhel-8-1/firefox',
-            'rhel-8-1/edge',
+        'rhel-8.2': [
+            'rhel-8-2/chrome',
+            'rhel-8-2/firefox',
+            'rhel-8-2/edge',
```

Comment:
There actually is no "rhel-8.2" composer branch, and "rhel-8.1" did not exist either. "rhel-8.0" still exists (but is largely neglected now, I think we can just delete it). I would recommend to just drop this paragraph completely. We can reintroduce it once we actually need and have a stable branch again, WDYT?